### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by mpujolWolox**
## Description
- Main layout styling

## Screenshoots
image

image

## Trello Card
- https://trello.com/c/tDXspxap

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
   margin: 0 auto;
-  max-width: 1000px;
+  max-width: 1200px;
 
   .tech {
-    display: inline-block;
     padding: 10px;
-    vertical-align: top;
-    width: 24%;
+    width: 25%;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Try not to use percentages, they will make the elemet to loose its shape. Use fixed values in px

File: scss/tech.scss [hunk 2]
```diff
@@ -54,7 +53,7 @@
     }
 
     @media screen and (max-width: 900px) {
```
Comment (mmamaniwolox) [inline scss/tech.scss]: Do media queries out of this css class, it should be general for all classes. You should need only one, we usually use 1000px as breakpoint.

File: scss/tech.scss (hunk 1 above)
Comment (mpujolWolox) [reply scss/tech.scss]: Done

File: scss/styles.scss [hunk 3]
```diff
@@ -3,3 +3,13 @@
 * {
   font-family: 'Roboto', sans-serif;
 }
+
+.flex-body {
+  display: flex;
+  flex-direction: column;
+
+  main {
```
Comment (SKOLZ) [inline scss/styles.scss]: use classes instead of html tags

File: scss/styles.scss [hunk 4]
```diff
@@ -3,3 +3,13 @@
...
+    margin: 0 auto;
+    width: 90%;
```
Comment (SKOLZ) [inline scss/styles.scss]: I think you should set width: 100% and a max-width in pixels to define the container's size. using 90% generates different spacing and widths depending on the screen and it's better to keep at a max size instead and use padding to generate this separation from the borders.

**Pull request merged**